app/frontend/src/features/profile/view/ProfilePage.tsx

```diff
@@ -39,9 +39,10 @@ const useStyles = makeStyles((theme) => ({
     textDecoration: "none",
   },
   root: {
+    paddingTop: theme.spacing(3),
     [theme.breakpoints.up("md")]: {
+      paddingTop: 0,
       display: "flex",
-      justifyContent: "space-around",
```

Comment:
I'm not at my desktop - is the profile page still centered when this is removed?

Reply:
It is